subsys/bluetooth/mesh/light_ctrl_srv.c

```diff
@@ -1320,13 +1329,18 @@ static int light_ctrl_srv_start(struct bt_mesh_model *mod)
 		store(srv, FLAG_STORE_STATE);
 		break;
 	case BT_MESH_ON_POWER_UP_ON:
-		ctrl_disable(srv);
+		if (atomic_test_bit(&srv->flags,
+		FLAG_CTRL_SRV_MANUALLY_ENABLED)) {
```

Comment:
If you have enabled the C/C++ extension in VS Code, you should be able to auto format the current selection with Ctrl+K, Ctrl+F. This looks a bit weird

Reply:
Done